sdk/storage/storage-file-share/src/utils/utils.common.ts

```diff
@@ -515,3 +515,48 @@ export function httpAuthorizationToString(
 ): string | undefined {
   return httpAuthorization ? httpAuthorization.scheme + " " + httpAuthorization.value : undefined;
 }
+
+/**
+ * Get URL path and query from an URL string.
+ *
+ * @param url - Source URL string
+ */
+export function getURLPathAndQuery(url: string): string {
+  const urlParsed = URLBuilder.parse(url);
+  const pathString = urlParsed.getPath();
+  if (!pathString) {
```

Comment:
When pathString is empty string, this will throw error. Please check if it's expected.

Reply:
Updated. We don't need this method anymore.